tests/queries/0_stateless/01780_column_sparse_tuple.sql

```diff
@@ -0,0 +1,43 @@
+DROP TABLE IF EXISTS sparse_tuple;
```

Comment:
Maybe we can add some test configuration that would switch all columns to sparse? Just temporarily, to find more bugs.

Reply:
I've already ran tests with all full columns since 1df3e26 and until some commit, and found several bugs after it. However, it's good to add separate check for some time.